mirgecom/eos.py

```diff
@@ -91,10 +93,18 @@ def sound_speed(self, cv: ConservedVars):
         """Get the gas sound speed."""
         raise NotImplementedError()
 
-    def gas_const(self, cv: ConservedVars = None):
+    def gas_const(self, cv: ConservedVars):
         r"""Get the specific gas constant ($R_s$)."""
         raise NotImplementedError()
 
+    def heat_capacity_cp(self, cv: ConservedVars):
+        r"""Get the specific heat capacity at constant pressure ($C_p$)."""
+        raise NotImplementedError()
```

Comment:
Minor comment, but this is a similar point that we discussed for the boundary condition interface. Why not make `GasEOS` an abstract class and simply use `@abstractmethod`? Like this:
```suggestion
    @abstractmethod
    def heat_capacity_cp(self, cv: ConservedVars):
        r"""Get the specific heat capacity at constant pressure ($C_p$)."""
```

Not functionally different, just cleaner/slicker IMO.